### DISCUSSION TITLE
osu!lazer tournament client not showing score and chat

### DISCUSSION THREAD

**Discussion by giuuulio** [ppy/osu#21055] [Q&A] [answered]
Hello, I've been having issues with lazer tournament client not showing the score or the chat in the "Gameplay" tab.
Initially I was sure the issue was that i couldn't connect the ipc.txt file because the lazer file explorer was literally not showing up and I had no way of telling it where to find the file.
Only earlier today (for some reason that I really don't know) the file explorer showed up and I could select the path where I have the other tournament client and the empty ipc.txt installed. This didn't change anything though, the score and the chat are still not showing up. I tried with a dummy lobby and just like last time only the other client was actually connected to the lobby and was receiving the messages from the chat and displaying the score.
Please help

I found this file in the logs of the osu!lazer tournament client installation folder:
[legacy-ipc.log](https://github.com/ppy/osu/files/9908582/legacy-ipc.log)
Doesn't seem quite right to me

**Comment by PercyDan54** [comment, marked as answer]
I believe you need to have a CuttingEdge version for that to work.

**Reply by giuuulio** [reply]
Yes that was the problem, thank you so much!<3
